Sources/Danger/BitBucketCloud.swift

```diff
@@ -16,7 +16,7 @@ public struct BitBucketCloud: Decodable {
     public let metadata: BitBucketMetadata
 
     /// The PR metadata
-    public let pr: PullRequest // swiftlint:disable:this identifier_name
+    public let pr: PullRequest 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 19;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)